src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2550,6 +2550,20 @@ namespace HVACUnitarySystem {
                 UnitarySystem(UnitarySysNum).MaxNoCoolHeatAirVolFlow / UnitarySystem(UnitarySysNum).MaxHeatAirVolFlow;
         }
 
+        // initialize idle air flow rate variables in case these are needed in multi-speed heating coil sizing when multi-speed cooling coil does not
+        // exist
+        // the multi-speed coils will overwrite this data and these variablse are only used for multi-speed coils in function SetOnOffMassFlowRate
+        if (MultiOrVarSpeedCoolCoil(UnitarySysNum) || MultiOrVarSpeedHeatCoil(UnitarySysNum)) {
```

Comment:
If there are no hold-ups I'll fix this typo in another branch. Otherwise I'll fix if we agree something in this branch needs changing.

Reply:
See above, the results look wrong here.

Reply:
Sorry, forgot to answer that one. I thought they were wrong too when I fixed this (because I used 0.25 * 1.298 and something happened, can't recall now what that was). Then I read the IO and found it's a fraction of the low speed flow rate. So this means the no load flow rate could never be greater than the lowest speed. Not sure anyone would ever want that but it does limit flexibility. So it's 0.25 * 0.3245 = 0.081125. So yes, the results look wrong, even after this explanation, but they are correct.

Reply:
That's not how I read the I/O Ref:
> The no load air flow rate will be calculated as this fraction multiplied by the minimum of the cooling and heating supply air flow rate.

To me, that means it's the fraction of either Heating Supply Air Flow Rate or Cooling Supply Air Flow Rate from the AirloopHVAC:UnitarySystem object, whichever is smaller.

Reply:
OK, maybe I read that too quickly. But I had it as a fraction of the max (i.e., the min of heating and cooling max) before and something happened. I think it was that I tried autosize and it gave me a no load flow rate equal to the max. So the autosized multispeed coil operated at full flow when there was no load yet operated at lower flows when there was a load. That just seemed wrong since most files are autosized. Confirm you want me to change this before I go to that trouble (of making sure it's the same everywhere).

Reply:
Well, the default fraction is 1.0 which is probably not the best choice, but it's been that way. 
```
UnitarySystemPerformance:Multispeed,
  N3 , \field No Load Supply Air Flow Rate Ratio
       \type real
       \minimum 0
       \maximum 1
       \default 1
       \note Used to define the no load operating air flow rate when the system fan
       \note is specified to operate continuously.
```
I'm not following your comments. It shouldn't matter if the coil is autosized or not, this is at the unitary system level. Now, if there are example files that have a blank in this field or a 1.0, then, perhaps those should be change, but I don't know if that's relevant here.

As it stands now, the defect file is autosizing the no load flow rate to 0.32450 (25% of full flow) as expected.  It's just not using that during operation.

Reply:
OK so I'll change it, and the typo. I agree with your assessment. That's the only change, right?

Reply:
I think so. There is a strange test failure [here](https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-e17266cba39a9a4fe68cfcb5ff574472db47fd3e-x86_64-Linux-Ubuntu-14.04-gcc-4.8-IntegrationCoverage-Debug.html) in integration.OutdoorAirUnitwithAirloopHVAC but hopefully that goes away when the missing line fix is merged in.  Waiting for CI to finish that and then I'll merge it.

Reply:
I can't think of a way to default this logically. I could make it the min of min speed flows or make it a required field (don't like that idea). I just don't like the way a blank or autosized field is handled.

Reply:
I'm not following. Are you talking about No Load Supply Air Flow Rate Ratio? Just let it be 1.0. It's easy to set to something else.